rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskQueryFilterParameter.java

```diff
@@ -28,6 +28,7 @@ public class TaskQueryFilterParameter implements QueryParameter<TaskQuery, Void>
   /** Filter by what the task id shouldn't be. This is an exact match. */
   @JsonProperty("task-id-not")
   private final String[] taskIdNotIn;
+
```

Comment:
Did the formatter add these empty spaces? Can you change it back, also for the other changes made by the formatter

Reply:
I setup the formatter as specified in the documentation, and it added those spaces, so they are probably fine, I guess. Should I really change them back? @holgerhagen @gitgoodjhe

Reply:
If the formatter added it, it's fine